script.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+
+log_file=""
+method=1
+country_flag=false
```

Comment:
Why did you set the `log_file` to empty? It would be nice to have a default file path. Please add the ability to run the script without `-f` and in this case use the default path